helpers/guilds/getBans.ts

```diff
@@ -4,8 +4,18 @@ import { DiscordBan } from "../../types/discord.ts";
 import { User } from "../../transformers/member.ts";
 
 /** Returns a list of ban objects for the users banned from this guild. Requires the BAN_MEMBERS permission. */
-export async function getBans(bot: Bot, guildId: bigint) {
-  const results = await bot.rest.runMethod<DiscordBan[]>(bot.rest, "get", bot.constants.endpoints.GUILD_BANS(guildId));
+export async function getBans(bot: Bot, guildId: bigint, options?: GetBans) {
+  const queries = [];
+
+  if (options?.limit) queries.push(`limit=${options.limit}`);
+  if (options?.after) queries.push(`after=${options.after}`);
+  if (options?.before) queries.push(`before=${options.before}`);
+
+  const results = await bot.rest.runMethod<DiscordBan[]>(
+    bot.rest,
+    "get",
+    `${bot.constants.endpoints.GUILD_BANS(guildId)}${queries.length ? `?${queries.join("&")}` : ""}`,
+  );
```

Comment:
```suggestion
  let url = bot.constants.endpoints.GUILD_BANS(guildId);

  if (options) {
    url += "?";

    if (options.limit) url += `limit=${options.limit}&`;
    if (options.after) url += `after=${options.after}&`;
    if (options.before) url += `before=${options.before}`;
  }

  const results = await bot.rest.runMethod<DiscordBan[]>(bot.rest, "get", url);
```

Reply:
@itohatweb
Is `url?` valid? (everything is optional)
Is `?&after` valid? (everything is optional)

Reply:
I discussed this design with skillz.
I proposed this way of doing and therefore yes it is valid.